data_safe_haven/pulumi/dynamic/ssl_certificate.py

```diff
@@ -138,12 +140,14 @@ def delete(self, id_: str, props: dict[str, Any]) -> None:
                 key_vault_name=props["key_vault_name"],
             )
         except Exception as exc:
-            msg = f"Failed to delete SSL certificate [green]{props['certificate_secret_name']}[/] for [green]{props['domain_name']}[/].\n{exc!s}"
+            msg = (
+                f"Failed to delete SSL certificate [green]{props['certificate_secret_name']}[/]"
```

Comment:
E.g. see here where necessary whitespace is missing

Reply:
Fixed in 19bad5a47040fd59008289caa49c40cd31bacd8f